### PR TITLE
Add configs for Presto

### PR DESCRIPTION
**Pull request by lu4nm3**
This is pending on this PR (lyft/flyteplugins#69)

### CODE REVIEW

File: kustomize/overlays/sandbox/propeller/plugins/presto/propeller-patch.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This file is only for volume mounts.  The configmap itself that's being mounted is sufficiently different that
```
Comment (wild-endeavor) [inline kustomize/overlays/sandbox/propeller/plugins/presto/propeller-patch.yaml]: I wrote this comment at one point I think.  At the time, it made sense.  At this point, I have no idea what it means.

**Comment by wild-endeavor** [comment]
does it make kustomize?

**Comment by lu4nm3** [comment]
One of the tests is failing but  I can't tell if it's due to my changes.

**Comment by mweagle** [comment]
Hi @lu4nm3  - is this PR still valid?

**Comment by lu4nm3** [comment]
Closing this PR for now. Since the open source Presto client is not there, it doesn't make sense to include these config changes.

**Pull request closed without merging**